src/components/templates/ExternalRoom/ExternalRoom.scss

```diff
@@ -1,41 +1,92 @@
 @import "scss/constants";
 
+$border-radius: 22px;
+$spacing: 20px;
+
 .ExternalRoom {
   height: 100%;
-  width: 80%;
   display: flex;
+  background-position: center;
+  background-size: cover;
   align-items: center;
   justify-content: center;
-  margin: 0 auto;
+  flex-direction: column;
 
   &__message {
-    padding: 60px;
     display: flex;
     flex-direction: column;
+    justify-content: center;
     align-items: center;
-    justify-content: space-evenly;
-    background: $light-grey;
-    box-shadow: box-shadow--large(0.5);
-    border-radius: $border-radius--xl;
+    margin: $spacing--xxl 0;
+    font-family: Rubik;
+    font-weight: $font-weight--300;
+    font-size: $font-size--xl;
   }
 
   &__content {
     display: flex;
     flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    margin-top: $spacing--xxl;
+    max-height: 80vh;
+    border-radius: 20px;
+    background: $dark;
   }
 
-  &__header {
-    font-family: Rubik;
+  &__venue-container {
+    padding: $spacing;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+  }
+
+  &__venue-title {
+    font-weight: $font-weight--700;
+    font-size: $font-size--xxl;
+  }
+
+  &__venue-subtitle {
     font-weight: $font-weight--300;
-    font-size: $font-size--xl;
+    font-size: $font-size--lg;
   }
 
-  &__link {
-    font-family: Rubik;
-    font-style: normal;
-    font-weight: $font-weight--500;
+  &__venue-details {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: $spacing;
+  }
+
+  &__venue-logo {
+    flex-shrink: 0;
+    height: 180px;
+    width: 180px;
+    border-radius: $border-radius;
+    background-color: $black;
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__venue-description {
+    @include scrollbar;
+    padding: 10px $spacing;
```

Comment:
how about `$spacing/2 $spacing`

Reply:
It's just a coincidence that the first one is `/2` of the `$spacing` value. If $spacing changes we don't want the other one to change as well.

Reply:
how about the `10px` snapping to one of those `$spacing-` constants? could be 8 or 12 I guess